examples/bootstrap

```diff
@@ -29,8 +30,9 @@ nlbuild_autotools_stem="third_party/nlbuild-autotools/repo"
 
 # Establish some key directories
 
-srcdir=`dirname ${0}`
-abs_srcdir=`pwd`
+srcdir="$(dirname "${0}")"
```

Comment:
when using `x=$()`, there's no need for `""`

Reply:
Thanks, though generally harmless, yes?

Reply:
yeah, just looks bleh ;)